onboarding/views.py

```diff
@@ -50,6 +59,7 @@ def tracks_view(request):
 
 @login_required
 def user_track_view(request, user_id=None, track_id=None):
+    
```

Comment:
Change the following to:
entry = UserTrackBridge.objects.filter(user=user_id, track=track_id).exists()

Reply:
I will change it

Reply:
You didn't ask why :)

Reply:
I will next meeting  Inshaalah, but I think because we getting the data from the form so that it could be empety and we will need the None keywords.

Reply:
Cool

Reply:
is it the answer? if you have time to answer me

Reply:
Let's explain it in our next session, put it as a note

Reply:
Okay, thank you our hero